ducktape/cluster/vagrant.py

```diff
@@ -62,6 +62,10 @@ def __init__(self, *args, **kwargs):
             cluster_json["nodes"] = nodes
             json.dump(cluster_json, open(cluster_file, 'w+'), indent=2, separators=(',', ': '), sort_keys=True)
 
+        # Release any ssh clients used in querying the nodes for metadata
```

Comment:
Not sure I understand why we need this -- what would have opened an SSH connection at this point?